### PR TITLE
Test caching of astrometry and using libcfitsio packages

### PR DESCRIPTION
**Pull request by jamessynge**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -49,13 +50,21 @@ addons:
     - libbz2-dev
     - swig
     - cfitsio-dev
+    - cfitsio
+    - cfitsio*
```
Comment (wtgee) [inline .travis.yml]: Can the cfitsio* just be one line? And looks like this is failing.

File: .travis.yml [hunk 2]
```diff
@@ -8,8 +8,9 @@ python:
 env:
     - PANDIR=$HOME POCS=$TRAVIS_BUILD_DIR PANUSER=$USER ARDUINO_VERSION=1.8.1
 before_install:
+    - pwd
```
Comment (wtgee) [inline .travis.yml]: Is this just to see in logs?

File: .travis.yml [hunk 3]
```diff
@@ -49,13 +50,21 @@ addons:
...
-  - tar zxvf astrometry.net-0.72.tar.gz
-  - cd astrometry.net-0.72 && make && make py && make install INSTALL_DIR=$PANDIR/astrometry
+  - if [ ! -d $PANDIR/astrometry/bin || ! -d $PANDIR/astrometry/data ] ; then \
+      mkdir -p $PANDIR/astrometry/data && \
+      wget http://astrometry.net/downloads/astrometry.net-0.72.tar.gz && \
+      tar zxvf astrometry.net-0.72.tar.gz && \
+      cd astrometry.net-0.72 && \
+      make && \
+      make py && \
+      make install INSTALL_DIR=$PANDIR/astrometry && \
```
Comment (wtgee) [inline .travis.yml]: Dangling `&&`

**Comment by jamessynge** [comment]
Closing in favor of other changes.

**Pull request closed without merging**